### PR TITLE
Feature/segun/user dashboard rating

### PR DESCRIPTION
**Pull request by Segun-Ade**
#### What does this PR achieve?
This PR implements the logic for rating a vendor service
#### Any background context?
The Value for any rate is saved in the state for now as the logic for a successful order is being fixed by @ChineduOguejiofor. When that is ready I'll add a saga to handle the request 
#### Screenshots:
DeepinScreenshot_20200110163515

### CODE REVIEW

File: src/components/userDashboard/components/overview/components/Dashboard.jsx [hunk 1]
```diff
@@ -77,7 +73,7 @@ const topVendorColumn = [
         dataIndex: 'rating',
         render: rating => (
             <span>
-                <Rate allowHalf defaultValue={rating} />
+                <Rate disabled allowHalf defaultValue={rating} />
             </span>
```
Comment (Way2nnadi) [inline src/components/userDashboard/components/overview/components/Dashboard.jsx]: is this `span` needed?

File: src/components/userDashboard/components/rating/components/rateVendor.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        setRateValue(x);
+    };
+    useEffect(() => {
+        handleRating(rateValue);
+    }, []);
+
+    return (
+        <Rate
+            className="rate"
+            onChange={handleRating}
```
Comment (Way2nnadi) [inline src/components/userDashboard/components/rating/components/rateVendor.jsx]: just pass `setRateValue` directly to onchange
```
onChange={setRateValue}
```

File: src/components/userDashboard/components/overview/components/Dashboard.jsx (hunk 1 above)
Comment (Segun-Ade) [reply src/components/userDashboard/components/overview/components/Dashboard.jsx]: Yes it is

File: src/components/userDashboard/components/rating/components/rateVendor.jsx (hunk 2 above)
Comment (Segun-Ade) [reply src/components/userDashboard/components/rating/components/rateVendor.jsx]: OK

File: src/components/userDashboard/components/overview/components/Cta.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        {CTA_BUTTON_STRING.map(button => {
+            const { classname, url, text } = button;
+            const links = `${path}/${url}`;
+            return (
+                <Link className={classname} to={links} key={url}>
+                    {text}
+                </Link>
+            );
+        })}
+    </div>
```
Comment (Way2nnadi) [inline src/components/userDashboard/components/overview/components/Cta.jsx]: use a fragment here instead of a div

Comment (Segun-Ade) [reply src/components/userDashboard/components/overview/components/Cta.jsx]: Noted

**Pull request merged**